### PR TITLE
Updating the minimum VS version

### PR DESCRIPTION
**Pull request by marcpopMSFT**
There have been issues with 8.0 running against 17.5

### CODE REVIEW

**Comment by marcpopMSFT** [comment]
Looks like I can't update this until 17.6 releases. I'll update the installer UI at least.

**Comment by marcpopMSFT** [comment]
@rainersigwald looks like you added the read of the minimummsbuildversion file in SDK. Looks like I can't update the minimum version until after the version releases from now on. Is that about right? I think I'm ok with that but just thought I'd confirm that. https://github.com/dotnet/sdk/blob/main/eng/Versions.props#L109

**Comment by rainersigwald** [comment]
I don't think that aspect of it was intentional. I think you could add the prerelease feed to unblock yourself for now:

https://github.com/dotnet/sdk/blob/f9b11911d4a0f53dcaba2c32f8b665d1f6ae2c4e/NuGet.config#L9

**Comment by marcpopMSFT** [comment]
@rainersigwald build still failed. Did I get the right feed?

**Comment by marcpopMSFT** [comment]
@rainersigwald per your comment, overriding the build value with 17.6.1 which comes from the feed. This forced a few additional changes that I'll want to run by Daniel.

**Review by rainersigwald — APPROVED** [review]

File: eng/Versions.props [hunk 1]
```diff
@@ -107,6 +107,7 @@
 
           This avoids the need to juggle references to packages that have been updated in newer MSBuild. -->
     <MicrosoftBuildPackageVersion Condition="exists('$(RepoRoot)\src\Layout\redist\minimumMSBuildVersion') and '$(DotNetBuildFromSource)' != 'true'">$([System.IO.File]::ReadAllText('$(RepoRoot)\src\Layout\redist\minimumMSBuildVersion').Trim())</MicrosoftBuildPackageVersion>
+    <MicrosoftBuildPackageVersion Condition="'$(MicrosoftBuildPackageVersion)' == '17.6.0'">17.6.1</MicrosoftBuildPackageVersion>
```
[the remark this answers is not in the document]
Comment (marcpopMSFT) [reply eng/Versions.props:111]: Added a comment and changed the references to use a property. I decided against removing the references as that's in the framework build and we may not know it's broken till we update the resolvers in VS in like August.

**Pull request merged**